bazel/envoy_internal.bzl

```diff
@@ -79,7 +79,8 @@ def envoy_copts(repository, test = False):
            }) + envoy_select_hot_restart(["-DENVOY_HOT_RESTART"], repository) + \
            _envoy_select_perf_annotation(["-DENVOY_PERF_ANNOTATION"]) + \
            envoy_select_google_grpc(["-DENVOY_GOOGLE_GRPC"], repository) + \
-           _envoy_select_path_normalization_by_default(["-DENVOY_NORMALIZE_PATH_BY_DEFAULT"], repository)
+           _envoy_select_path_normalization_by_default(["-DENVOY_NORMALIZE_PATH_BY_DEFAULT"], repository) + \
+           _envoy_select_legacy_codecs_in_test(["-DENVOY_USE_LEGACY_CODECS_IN_TEST"], repository)
```

Comment:
Can this only applied in relevant files (i.e. integration.cc)? Applying this to all files invalidates all cache.

Reply:
Done and tested

Reply:
Done and tested